.github/workflows/continuous-integration.yaml

```diff
@@ -61,7 +61,7 @@ jobs:
   int-tests:
     name: Integration tests
     needs: [ build ]
-    runs-on: ubuntu-latest
+    runs-on: sg-ubuntu-latest-4-cores
```

Comment:
is this really making any changes?

Reply:
Yes. After making this change there were no timeout errors.